unittest/gunit/keyring_vault/CMakeLists.txt

```diff
@@ -1,12 +1,25 @@
 INCLUDE(curl)
 
-IF(NOT GMOCK_FOUND)
+IF(NOT GMOCK_FOUND OR NOT WITH_KEYRING_VAULT)
     RETURN()
 ENDIF()
 
-CHECK_IF_LIB_FOUND(CURL "keyring_vault unit tests")
+MACRO (CHECK_IF_CURL_DEPENDS_ON_RTMP project_name)
+  EXECUTE_PROCESS(COMMAND ldd ${CURL_LIBRARY}
```

Comment:
BTW, @robgolebiowski 
I saw this message `The supplied CURL library depends on rtmp library` on Ubuntu Artful with one of the most recent `curl`. The plugin itself was built there but not the unit tests. May be this check is not accurate for Artful.

Reply:
Curl on artful depends on the rtmp library, my local builds also have this issue:

curl -V
curl 7.55.1 (x86_64-pc-linux-gnu) libcurl/7.55.1 OpenSSL/1.0.2g zlib/1.2.11 libidn2/2.0.2 libpsl/0.18.0 (+libidn2/2.0.2) librtmp/2.3
Release-Date: 2017-08-14
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP UnixSockets HTTPS-proxy PSL

Reply:
rtmp does not depend on the version of curl. It is just built with rtmp on some machines and on some it is not. From what I am aware of - rtmp has been built in with curl for quite some time.